iggy/src/consumer_offsets/get_consumer_offset.rs

```diff
@@ -9,6 +9,12 @@ use serde::{Deserialize, Serialize};
 use std::fmt::Display;
 use std::str::FromStr;
 
+/// `GetConsumerOffset` command that retrieves the offset of a consumer for a given partition from the server.
+/// It has additional payload:
+/// - `consumer` - the consumer that is storing the offset, either the regular consumer or the consumer group.
+/// - `stream_id` - unique stream ID (numeric or name).
+/// - `topic_id` - unique topic ID (numeric or name).
+/// - `partition_id` - partition ID on which the offset is stored. Has to be specified for the regular consumer. For consumer group it is ignored (use `None`).
 #[derive(Debug, Serialize, Deserialize, PartialEq)]
 pub struct GetConsumerOffset {
```

Comment:
Why there's no doc comments for fields, like in file below ?

Reply:
Good catch, will fix.